src/sentry/static/sentry/app/utils.tsx

```diff
@@ -225,21 +218,29 @@ export const buildTeamId = id => `team:${id}`;
 /**
  * Removes the organization / project scope prefix on feature names.
  */
-export function descopeFeatureName(feature) {
-  return typeof feature.match !== 'function'
-    ? feature
-    : feature.match(/(?:^(?:projects|organizations):)?(.*)/).pop();
+export function descopeFeatureName(feature: string): string {
+  if (typeof feature.match !== 'function') {
+    return feature;
+  }
+
+  const results = feature.match(/(?:^(?:projects|organizations):)?(.*)/);
+
+  if (results) {
+    return results.pop()!;
+  }
+
+  return feature;
```

Comment:
Personal preference, but I think the original shorter version was easier to read (But I'm also biased, because I wrote it)

Reply:
Even this last bit I think could just be `return results ? results.pop() : feature`

Reply:
@EvanPurkhiser oh interesting. I'll revisit this. 

Typescript indicated that the `match()` can return `null`, which is why I rewrote it like this.

Reply:
Ahhh, yeah makes sense.